internal/providers/terraform/aws/route53_record.go

```diff
@@ -1,107 +1,25 @@
-		return &schema.Resource{
-			Name:      d.Address,
-			NoPrice:   true,
-			IsSkipped: true,
-		}
-	}
-
-	costComponents := []*schema.CostComponent{}
-	limits := []int{1000000000}
-
-	var numbOfStdQueries *decimal.Decimal
-	if u != nil && u.Get("monthly_standard_queries").Exists() {
-		numbOfStdQueries = decimalPtr(decimal.NewFromInt(u.Get("monthly_standard_queries").Int()))
-		stdQueriesTiers := usage.CalculateTierBuckets(*numbOfStdQueries, limits)
-
-		if stdQueriesTiers[0].GreaterThan(decimal.Zero) {
-			costComponents = append(costComponents, queriesCostComponent("Standard queries (first 1B)", "DNS-Queries", "0", &stdQueriesTiers[0]))
-		}
-
-		if stdQueriesTiers[1].GreaterThan(decimal.Zero) {
-			costComponents = append(costComponents, queriesCostComponent("Standard queries (over 1B)", "DNS-Queries", "1000000000", &stdQueriesTiers[1]))
-		}
-	} else {
-		var unknown *decimal.Decimal
-
-		costComponents = append(costComponents, queriesCostComponent("Standard queries (first 1B)", "DNS-Queries", "0", unknown))
-	}
-
-	var numbOfLBRQueries *decimal.Decimal
-	if u != nil && u.Get("monthly_latency_based_queries").Exists() {
-		numbOfLBRQueries = decimalPtr(decimal.NewFromInt(u.Get("monthly_latency_based_queries").Int()))
-		lbrQueriesTiers := usage.CalculateTierBuckets(*numbOfLBRQueries, limits)
-
-		if lbrQueriesTiers[0].GreaterThan(decimal.Zero) {
-			costComponents = append(costComponents, queriesCostComponent("Latency based routing queries (first 1B)", "LBR-Queries", "0", &lbrQueriesTiers[0]))
-		}
+	var aliasType *string
 
-		if lbrQueriesTiers[1].GreaterThan(decimal.Zero) {
-			costComponents = append(costComponents, queriesCostComponent("Latency based routing queries (over 1B)", "LBR-Queries", "1000000000", &lbrQueriesTiers[1]))
-		}
-	} else {
-		var unknown *decimal.Decimal
-
-		costComponents = append(costComponents, queriesCostComponent("Latency based routing queries (first 1B)", "LBR-Queries", "0", unknown))
-	}
-
-	var numbOfGeoQueries *decimal.Decimal
-	if u != nil && u.Get("monthly_geo_queries").Exists() {
-		numbOfGeoQueries = decimalPtr(decimal.NewFromInt(u.Get("monthly_geo_queries").Int()))
-		geoQueriesTiers := usage.CalculateTierBuckets(*numbOfGeoQueries, limits)
-
-		if geoQueriesTiers[0].GreaterThan(decimal.Zero) {
-			costComponents = append(costComponents, queriesCostComponent("Geo DNS queries (first 1B)", "Geo-Queries", "0", &geoQueriesTiers[0]))
-		}
-
-		if geoQueriesTiers[1].GreaterThan(decimal.Zero) {
-			costComponents = append(costComponents, queriesCostComponent("Geo DNS queries (over 1B)", "Geo-Queries", "1000000000", &geoQueriesTiers[1]))
-		}
-	} else {
-		var unknown *decimal.Decimal
-
-		costComponents = append(costComponents, queriesCostComponent("Geo DNS queries (first 1B)", "Geo-Queries", "0", unknown))
+	if len(d.References("alias.0.name")) > 0 && d.References("alias.0.name")[0].Type != "aws_route53_record" {
```

Comment:
Although this is ok from a migration point of view but could you please explain to me what comparing a `Type` to a string mean?

Reply:
`.Type` on resource_data is a string that stores the Terraform resource type (https://github.com/infracost/infracost/blob/master/internal/schema/resource_data.go#L12)